README.md

```diff
@@ -40,4 +40,10 @@ The calculator uses a default style "DefaultStyle.css". You could create a copy
 You can embed the mortgage calculator into any web page. Simply insert the following HTML.
 
     <div id="mortgage-calculator-react"></div>
-    <script src="https://www.fastforma.com/mortgage-calculator-react.js"></script>
\ No newline at end of file
+    <script src="https://www.fastforma.com/mortgage-calculator-react.js"></script>
+    
+## Additional props
+You can ovverride default props.
+
+    
+    <MortgageCalculator price={125000} downPayment={10000} interestRate={0.065} months={5} taxRate={2} insuranceRate={1} mortgageInsuranceEnabled={false} additionalPrincipal={12500} />
```

Comment:
taxRate and insurance rate should probably be less than 1, since they are percentages.

additionalPrincipal is not yet supported as an override prop. Can be added pretty easily though.